examples/rebar3/src/rebar3.erl

```diff
@@ -0,0 +1,7 @@
+-module(rebar3).
```

Comment:
Slightly confusing example module name. It may be mistaken for the tool with the same name. How about my_example or something like that?

Reply:
rebar3_example